migrate/src/main/scala/migrate/utils/FileUtils.scala

```diff
@@ -11,18 +13,28 @@ import migrate.AbsolutePath
 
 private[migrate] object FileUtils {
   final def listFiles(base: AbsolutePath): Seq[AbsolutePath] =
-    listFiles(base.toFile, true).map(AbsolutePath.from)
+    listFiles(base.toFile, true).map(AbsolutePath.from).filter(_.value.endsWith(".scala"))
 
-  private def listFiles(base: File, recursive: Boolean = true): Seq[File] = {
-    val (files, directories) = base.listFiles().toSeq.partition(_.isFile)
-    files ++ {
-      if (recursive) directories.flatMap(listFiles(_, recursive))
-      else Seq.empty
+  final def writeFile(file: AbsolutePath, content: String): Try[Unit] =
+    Try {
+      val bw = new BufferedWriter(new FileWriter(file.toFile))
+      bw.write(content)
+      bw.close()
     }
-  }
+
+  private def listFiles(base: File, recursively: Boolean = true): Seq[File] = Try {
+    def listDir(dir: File): List[File] =
+      if (dir.isDirectory) {
+        Option(dir.listFiles).map(_.toList.flatMap(f => if (recursively) listDir(f) else List(f))).getOrElse(List())
+      } else {
+        List(dir)
+      }
+    listDir(base).filter(_.isFile).sorted
+  }.getOrElse(List())
```

Comment:
Not used anymore and I don't think we will need it.

Reply:
ok